src/rnp/rnpcfg.c

```diff
@@ -497,6 +576,105 @@ rnp_cfg_get_defkey(rnp_cfg_t *cfg, rnp_params_t *params)
     } else {
         params->defkey = strdup(userid);
     }
+}
 
-    return RNP_OK;
+/**
```

Comment:
I think comments should be in header files (as source files are not always delivered with library)